### ISSUE TITLE
Doesn't work with flatpaked apps

### ISSUE THREAD

**Issue by E100Beta** [kozec/sc-controller#474]
I tried running Steam and PCSX2 Flatpak, and both didn't see steam controller with scc-daemon running. Tried giving apps access to /dev, /home/$USER/.config/scc/daemon.socket, all accessible buses, triple-checked udev rules, nothing worked.

Can this be a bug on sc-controller's side?
Does anyone know what can help me with making things work?

PS: non-flatpaked Steam worked just fine on my setup.

**Comment by kozec** [comment]
> Can this be a bug on sc-controller's side?

More likely it's a feature of flatpak :)

Check if giving access to /dev/input works, and if not, try filling bug with https://github.com/flatpak/flatpak.

**Comment by E100Beta** [comment]
Yes, Steam has access to /dev/input, and executing `cat /dev/input/event18` (it was num corresponding to Xbox pad after starting scc-daemon) from inside the flatpak container gives me gibberish in response to pressing buttons on controller, so technically it works.

So the problem is apps don't recognize `/dev/inpur/event*` as a joystick?

**Comment by kozec** [comment]
What game are you trying to run like that?

**Comment by E100Beta** [comment]
Currently, Dead Cells and Sunless Skies, but I can try multiple other games tomorrow (both games detecting scc normally through simple chroot).

Also, if it helps, I tried udev rules from both of these sources:
https://gitlab.com/Fabish/game-devices-udev/blob/master/71-Valve-Controllers.rules
https://steamcommunity.com/app/353370/discussions/2/1735465524711324558/

I am sorry if it seems like I persist after you told me to report this to flatpak - I just want to understand, what may be the core issue I should report here. What may be the difference between `event*` created by sc-controller and by original device? Does it not tell the system something flatpak relies on?
